app/models/framework_response/multiple_item_object.rb

```diff
@@ -56,8 +57,9 @@ def response_objects
     end
 
     def validate_responses_type
-      unless responses_type_valid?
+      unless responses.blank? || responses_type_valid?
         errors.add(:responses, 'is incorrect type')
+        raise ActiveModel::ValidationError, self
```

Comment:
Same comment applies here I think too - as this exception is potentially raised via the constructor, perhaps consider throwing the custom exception instead. Also no need to append to `errors`.